src/Ionide.ProjInfo/Types.fs

```diff
@@ -68,6 +68,13 @@ module Types =
         Items: ProjectItem list
         Properties: Property list
         CustomProperties: Property list
+        /// This contains all Project Properties seen during evaluation
+        AllProperties: Map<string, Set<string>>
+        /// This contains all Project Items seen during evaluation with their values and associated metadata
+        AllItems: Map<string, Set<string * Map<string, string>>>
```

Comment:
Allow consumers to just get at anything and implement various feature without ProjInfo needing changed. I'm not super happy needing new names because we have "Properties"/"Items" already but I didn't wanna break those at this point.